### ISSUE TITLE
Syntax notes shows same snippets for right and wrong usages

### ISSUE THREAD

**Issue by yiochen** [styled-components/styled-components-website#330]
on https://www.styled-components.com/docs/tooling#template-literal-style-and-indentation
The right and wrong usages looks the same
![image](https://user-images.githubusercontent.com/5957726/46241098-d0aed780-c366-11e8-99e4-8c23b3268c82.png)

**Comment by mxstbr** [comment]
Haha damn, I think prettier auto-formatted all of these to the correct syntax! /cc @chinesedfan

**Comment by chinesedfan** [comment]
@mxstbr Right. It was caused by https://github.com/styled-components/styled-components-website/commit/d37500db0a9173b977f31ccdb0366825317522b8.

**Comment by joaopedrovc** [comment]
I will fix it.

**Comment by chinesedfan** [comment]
@joaopedrovc Sorry, #333 has tired to fix.
